packages/subgraph/src/mapping.ts

```diff
@@ -214,6 +207,19 @@ export function handleSetContinuousSwap(event: LOG_SET_FLOW): void {
   updatePoolDayData(event, 'continuous');
   updatePoolHourData(event, 'continuous');
   updateTokenDayData(tokenIn, event, 'continuous');
+
+  if(event.params.tokenRateIn.equals(BigInt.fromI32(0))) {
+    store.remove('ContinuousSwap', swap.id);
+    return;
+  }
+
+  swap.timestamp = event.block.timestamp;
+  swap.transaction = event.transaction.hash.toHex();
+  swap.minOut = convertTokenToDecimal(event.params.minOut, tokenOut.decimals);
+  swap.maxOut = convertTokenToDecimal(event.params.maxOut, tokenOut.decimals);
+  swap.rateIn = convertTokenToDecimal(event.params.tokenRateIn, tokenIn.decimals);
+  swap.currentRateOut = BigInt.fromI32(0).toBigDecimal();
```

Comment:
You set the rate out but you neglect to update the `totalOutUntilLastSwap` amount and `timestampSinceLastSwap`.